src/parser/shared/modules/items/FoodChecker.js

```diff
@@ -41,19 +41,26 @@ class FoodChecker extends Analyzer {
       }
     }
   }
-  get foodSuggestionThresholds() {
+  get higherFoodSuggestionThresholds() {
     return {
       actual: this.higherFoodUp,
       isEqual: false,
       style: 'boolean',
     };
   }
+  get isPresentFoodSuggestionThresholds() {
+    return {
+      actual: this.higherFoodUp || this.lowerFoodUp,
+      isEqual: false,
+      style: 'boolean',
+    };
+  }
   suggestions(when) {
     let suggestionText = 'You did not have any food buff combat started. Having the right food buff during combat is an easy way to improve performance.';
     if (!this.higherFoodUp && this.lowerFoodUp) {
       suggestionText = 'You did not have the best food active when starting the fight. Using the best food available is an easy way to improve performance.';
     }
-    when(this.foodSuggestionThresholds)
+    when(this.higherFoodSuggestionThresholds)
       .addSuggestion((suggest) => {
         return suggest(suggestionText)
           .icon(SPELLS.BOUNTIFUL_CAPTAIN_FEAST_AGI.icon)
```

Comment:
Should probably add an average or major issue for no food at all?

Reply:
good idea

Reply:
http://localhost:3000/report/kXJdwmhqD3z4frna/1-Heroic+Taloc+-+Kill+(7:09)/12-Deathsblad%C3%A9

![image](https://user-images.githubusercontent.com/9600587/49703850-5f966680-fbd8-11e8-9376-006e3ed539bd.png)